protecode/model.py

```diff
@@ -68,13 +68,15 @@ def cve(self):
     def cve_severity_str(self):
         return str(self.raw.get('vuln').get('cvss'))
 
+    def has_triages(self):
+        return self.raw.get('triage') is not None
+
     def cve_major_severity(self) -> int:
-        if self.cve_severity_str():
+        if self.cve_severity_str() and not self.has_triages():
```

Comment:
this filtering should not be hard-coded. either perform it in the caller determining the report or at least introduce a flag (defaulting to not to filter)